tests/pkunit_test.py

```diff
@@ -152,6 +153,10 @@ def test_pkre_convert():
     for s in ('A', b'A', r'A', u'A'):
         pkre(r, s)
 
+def test_xlsx_to_csv_conversion():
+    from pykern.pkunit import file_eq, data_dir
+
+    file_eq(expect_path='example.csv', actual_path=data_dir().join('example.xlsx'))
```

Comment:
Is this more what you had in mind @robnagler? Now this lives in pkunit_test.py and example.csv/xlsx live in pkunit_data. file_eq seems to be converting .xlsx and comparing that with example.csv correctly